packages/app-store/salesforce/lib/CrmService.ts

```diff
@@ -915,6 +826,51 @@ export default class SalesforceCRMService implements CRM {
     return this.getDominantAccountId(response.records as { AccountId: string }[]);
   }
 
+  private async getAccountBasedOnEmailDomainOfContacts(email: string) {
```

Comment:
Similar to `getAccountIdBasedOnEmailDomainOfContacts` but this returns the whole account object. I would like to migrate to this method but that'll involve refactoring in other places.